static-site/content/docs/07-visualisation/02-map-interpretation.md

```diff
@@ -0,0 +1,98 @@
+
+
+## Demes (Circles)
+
+Perhaps the most fundamental aspect of Nextstrain's map, the circles (demes) associate samples (tips on the tree) with locations on the map.
+The demes use different visual techniques to convey different pieces of information:
+
+![](./img/labelled-deme.png)
+_Figure 2. Examples of demes.
+B,C and D are representations of samples from the USA in [the Zika dataset](https://nextstrain.org/zika) with different choices of color-by._
+
+
+The **size (area) of the circle** is used to convey the number of samples associated with the current location.
+This should only be interpreted relative to other circles currently displayed on the map, and may change as you zoom into the map or subsample the data.
+For instance, in Fig 2A, the larger circle represents a deme with four times as many samples as the smaller one.
+As you filter the dataset, change the displayed time range, or zoom into the tree, the relative sizes of the circles will change according to what locations the in-view samples represent.
+
+Ancestral (inferred) nodes on the tree are _not_ represented by circles on the map.
+
+
+A circle's **color** conveys the metadata currently selected as the overall coloring of the dataset, and the way by which we convey this changes according to the type of color-scale employed.
+For instance, in Fig 2B the circle comprises samples from the USA and is therefore colored red (this is actually the circle labelled "1" in Figure 1). 
+In Fig 2C we have a discrete color scale representing authorship, and we convey this information by a pie chart with areas of each slice representing the proportion of samples in this location from each author.
+In Fig 2D we have a continuous color scale (sampling date) and convey this by a blending of colors -- the samples which make up this deme actually span Jun 2016 and Oct 2016.
+You can see the legend representing in currently selected metadata in either the top-left of the phylogenetic tree or the top-right of the map (see "3" in Figure 1).
+
+
+The **location** of circles represents the geographic resolution currently selected.
+Looking at Figure 1, we have a geo-resolution of Country (labelled "5") and so each circle represents an individual country.
+In the [zika dataset](https://nextstrain.org/zika) you can change this to "Region" and you'll see the map update accordingly.
+Each dataset is responsible for associating different locations with a GPS co-ordinate which we use as the center of the circle -- bear this in mind when you are interpreting the map and don't over-assign precision.
+For example, the circle representing the USA (labelled "1" in Figure 1) represents the entire country, not a specific location in Kansas!
+
+
+## Transmissions (Lines)
+
+These lines typically represent our best estimate of the movement of the pathogen over the selected time slice.
+For instance, the line indicated by "2" in Figure 1 indicates inferred movement of Zika virus from French Polynesia to Brazil.
```

Comment:
Not sure I have a solution, but I think the reference to Fig 1, which is now way high up on the page, is confusing, when Fig 3 is just below (and also has a line '2'). Maybe better to just switch to referencing Fig 3 entirely?